bower-publish.sh

```diff
@@ -7,6 +9,8 @@ if [ $CURRENT_BRANCH != 'master' ] ; then
   exit 0
 fi
 
+CHOSEN_VERSION=`git tag -l "*.*.*" | sed 's/^v//' | sort -V | tail -1`
```

Comment:
So I know this script is intended to be run on Travis (which by default is Ubuntu), but is this script ever intended to be run locally and/or anywhere else?  `sort -V` is not supported on OS X (or any BSD).

Worth double checking that it's even supported on Travis?  Their [environment docs](http://docs.travis-ci.com/user/ci-environment/) don't mention anything about coreutils.